layouts/partials/head.html

```diff
@@ -36,7 +25,7 @@
 {{- template "_internal/schema.html" . -}}
 {{- template "_internal/twitter_cards.html" . -}}
 {{- if hugo.IsProduction -}}
-{{ template "_internal/google_analytics_async.html" . }}
+{{ template "_internal/google_analytics.html" . }}
```

Comment:
Why this change? Is that adopted from upstream Docsy?

Reply:
> Why this change?

From [Use the Google Analytics Template](https://gohugo.io/templates/internal/#use-the-google-analytics-template) in the Hugo docs:

> When using Google Analytics v4 use `_internal/google_analytics.html`.

As for Docsy: the logic is different from the local change made in `head.html`.